### PR TITLE
Add initial version of process description and template for CSEP

### PR DESCRIPTION
**Pull request by strieflin**
See: #47

Signed-off-by: Sven Trieflinger <sven.trieflinger@de.bosch.com>

### CODE REVIEW

File: enhancements/NNNN-CSEP-template.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+<!-- TOC -->
+
+## Summary
+
+## Motivation
+
+### Goals
+
+### Non-Goals
+
```
Comment (sbckr) [inline enhancements/NNNN-CSEP-template.md:28]: I have an idea of what this section is about, but I'm not 100% sure. Can we assume that people know the intent of each section, or should there be at least a brief description of each?

File: enhancements/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+   create an issue to track the implementation of the proposal. Note that the
+   issue will go through the regular triage process and gets assigned a
+   priority. CSEPs should be developed in the public whenever possible as a
+   shared document in this folder. The name of the file should follow the
+   pattern `NNNN-short-title.md` where `NNNN` is the zero prefixed issue number
+   used to track the implementation of the CSEP from above.
+
+## Credits
+
+The CSEP process is heavily inspired by IETF RFCs, Kubernetes KEPs and Rust
```
Comment (sbckr) [inline enhancements/README.md:28]: Add links?

File: enhancements/NNNN-CSEP-template.md (hunk 1 above)
Comment (strieflin) [reply enhancements/NNNN-CSEP-template.md:28]: I think that is fine for now. As said, this process is heavily inspired by the respective process in other OS projects, in particular K8s KEPs. They also have no explanations in their template. What I did to understand what is expected was looking into concrete KEPs that "instantiate" the template. That worked out well for me. So I would suggest to see if that also works out in the CS community.

File: enhancements/README.md (hunk 2 above)
Comment (strieflin) [reply enhancements/README.md]: I have added links to the respective  "specification" documents.

**Review by sbckr — APPROVED** [review]
LGTM

**Pull request merged**